src/integrationtests/java/com/aws/greengrass/integrationtests/lifecyclemanager/GenericExternalServiceIntegTest.java

```diff
@@ -705,10 +705,10 @@ void GIVEN_service_starts_up_WHEN_startup_times_out_THEN_timeout_error_code_pers
         kernel.launch();
 
         assertTrue(serviceErroredLatch.await(15, TimeUnit.SECONDS));
-        assertThat(statusA.get().getStatusCode(), contains(ComponentStatusCode.STARTUP_TIMEOUT.toString()));
+        assertThat(statusA.get().getStatusCodes(), contains(ComponentStatusCode.STARTUP_TIMEOUT.toString()));
         assertThat(statusA.get().getStatusReason(),
                 containsString(ComponentStatusCode.STARTUP_TIMEOUT.getDescription()));
-        assertThat(statusB.get().getStatusCode(), contains(ComponentStatusCode.RUN_TIMEOUT.toString()));
+        assertThat(statusB.get().getStatusCodes(), contains(ComponentStatusCode.RUN_TIMEOUT.toString()));
```

Comment:
As we discussed offline, we currently don't have integ tests for shadow deployment because we want to test if any info in shadow responses carry over but in integ tests we would have to mock all shadow responses.

It would be better to test with a UAT